x/attribute/module.go

```diff
@@ -28,10 +29,11 @@ import (
 )
 
 var (
-	_ module.AppModule           = AppModule{}
-	_ module.AppModuleBasic      = AppModuleBasic{}
-	_ module.AppModuleSimulation = AppModule{}
-	// TODO[1760]: app-module: Add more assertions for the new types and clean up stuff no longer needed.
+	_ module.AppModuleBasic      = (*AppModule)(nil)
+	_ module.AppModuleSimulation = (*AppModule)(nil)
+
+	_ appmodule.AppModule       = (*AppModule)(nil)
+	_ appmodule.HasBeginBlocker = (*AppModule)(nil)
```

Comment:
Is there an `appmodule.HasEndBlocker` that we should be asserting here too?

Reply:
NM... missed that this was in a single module (and not the app as a whole). All good.